### ISSUE TITLE
Look into Custom Domain

### ISSUE THREAD

**Issue by solderq35** [solderq35/portfolio#18]
Can get a custom domain with Github Student Developer Pack which I have. 
- May be better for branding etc, and avoiding getting my site blocked by company VPN. 
- Although I'm also happy with my github pages domain for now. 
- Look into expenses etc, as I won't be on Github Student pack forever.

**Comment by solderq35** [comment]
probably won't fix in the forseeable future as this is not very important. Maybe look into buying personal domain after stable income

**Comment by solderq35** [comment]
edit - completed. see README for updates on how to do this
